pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -114,7 +125,19 @@ func (p *cadvisorStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 			// Special case for infrastructure container which is hidden from
 			// the user and has network stats.
 			podStats.Network = cadvisorInfoToNetworkStats("pod:"+ref.Namespace+"_"+ref.Name, &cinfo)
-			podStats.StartTime = metav1.NewTime(cinfo.Spec.CreationTime)
+			var startTime metav1.Time
+			for _, condition := range podStatus.Conditions {
+				if condition.Type == v1.PodScheduled && condition.Status == v1.ConditionTrue {
+					startTime = condition.LastTransitionTime
+					break
+				}
+			}
+			if startTime.IsZero() {
+				glog.V(3).Infof("Unable to find the start time for pod: %+v", ref)
+				continue
+			} else {
+				podStats.StartTime = startTime
```

Comment:
Don't need the `else` since you already have `continue` in line 137

Reply:
done